### ISSUE TITLE
nothing happens when trying to extract 

### ISSUE THREAD

**Issue by savinpvtmike** [Jayveer/Solideye#7]
i need help extracting stuff it seems not to do anything idk what im doing wrong i have tried decrypting stuff idk if i need other files idk if i need to use the dictionary.txt

**Comment by baaannnaaannn** [comment]
figured out?

**Comment by savinpvtmike** [comment]
yes i had a damaged disc when extracting the files so they were corrupted so i wound up downloading the game instead
